### PR TITLE
Do whitelisted worlds disabling properly

### PR DESCRIPTION
**Pull request by dordsor21**
//////////////////   
// PR's are NOT accepted on any branch other than dev-nightly.   
/////////////////

### CODE REVIEW

File: src/main/java/com/plugish/woominecraft/WooMinecraft.java [hunk 1]
```diff
@@ -175,12 +175,15 @@ public boolean check() throws Exception {
 				continue;
 			}
 
-			// If the user isn't in a white-listed world, commands will not run here.
-			if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {
+			//Check for enabled world whitelisting in config
+			if(getConfig().getBoolean("enable-world-whitelist")){
```
Comment (JayWood) [inline src/main/java/com/plugish/woominecraft/WooMinecraft.java]: At this point we'd be adding a config option for a config option. Probably not the best idea.

File: src/main/java/com/plugish/woominecraft/WooMinecraft.java [hunk 2]
```diff
@@ -175,12 +175,15 @@ public boolean check() throws Exception {
 				continue;
 			}
 
-			// If the user isn't in a white-listed world, commands will not run here.
-			if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {
+			//Check for enabled world whitelisting in config
+			if(getConfig().getBoolean("enable-world-whitelist")){
+				// If the user isn't in a white-listed world, commands will not run here.
+				if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {
```
Comment (JayWood) [inline src/main/java/com/plugish/woominecraft/WooMinecraft.java]: Work needs to be done on this, I agree, but overall I'm almost 100% positive there's a better way than stringing this together.

**Comment by JayWood** [comment]
Given the latest commit - white listed worlds should no longer be a problem.

https://github.com/WooMinecraft/WooMinecraft/commit/fc152d4df72d6056ebc1d8d703da745dd359ec9d

**Comment by dordsor21** [comment]
The commit you made won't make a difference, there's still no way to disable white-listed worlds. You still need to add them all manually

**Comment by JayWood** [comment]
@dordsor21 that's the plan, for now.

**Comment by dordsor21** [comment]
I'll maintain my own fork with that in then...because I'm _not_ adding 20+ worlds to the config

**Comment by JayWood** [comment]
@dordsor21 - See above, a better way of doing it is using `isSet` to see if the config is even present, instead of adding a secondary config option.

**Pull request closed without merging**